src/ZfrCors/Factory/CorsOptionsFactory.php

```diff
@@ -28,15 +27,17 @@
  * @license MIT
  * @author  Florent Blaison <florent.blaison@gmail.com>
  */
-class CorsOptionsFactory implements FactoryInterface
+class CorsOptionsFactory
 {
     /**
      * {@inheritDoc}
+     *
+     * @return CorsOptions
      */
-    public function createService(ServiceLocatorInterface $serviceLocator)
+    public function __invoke(ContainerInterface $container, $requestedName, $options = null)
```

Comment:
I suppose you'll need to also add the "createService" if you want to provide compatibility with SM 2.7. Confirm @weierophinney ?